raaghu-elements/src/rds-alert/rds-alert.css

```diff
@@ -27,4 +27,8 @@
 
 .z-index{
   z-index: 1;
+}
+
+.alert-left-border {
+  border-left: 5px solid var(--bs-alert-border-color); 
```

Comment:
@AkshayPotphode remove var(--bs-alert-border-color) and check is color is applying or not

Reply:
@Ketan-Khalate removed  var(--bs-alert-border-color) and checked, color is applying but it's too dark check once it is ok or need to change